### DISCUSSION TITLE
Using specific VAE for Dreambooth with custom model

### DISCUSSION THREAD

**Discussion by remybonnav** [TheLastBen/fast-stable-diffusion#1316] [Q&A]
Let say that I have a custom model and I need rhe sd 1.5 vae for it.
How can I add it for dreambooth training?
After converting ckpt to diffuser, it should be possible to add the VAE in the folder that was created for the model after conversion?

**Comment by TheLastBen** [comment]
Yes you can do that, first download the original V1.5, then the custom, after that, copy the content of the vae folder from the original to the custom one. the original v1.5 already contains the newest vae so no need to download it from hf.

**Comment by remybonnav** [comment]
Great,

Thanks a lot.
